homeassistant/components/ecovacs/strings.json

```diff
@@ -24,6 +24,47 @@
         "name": "Mop attached"
       }
     },
+    "sensor": {
+      "last_error": {
+        "name": "Last error"
+      },
+      "life_span_brush": {
+        "name": "Brush"
+      },
+      "life_span_filter": {
+        "name": "Filter"
+      },
+      "life_span_side_brush": {
+        "name": "Side brush"
+      },
+      "stats_area": {
+        "name": "Stats area"
+      },
+      "stats_time": {
+        "name": "Stats time"
+      },
+      "stats_total_area": {
+        "name": "Stats total area"
+      },
+      "stats_total_cleanings": {
+        "name": "Stats total cleanings"
+      },
+      "stats_total_time": {
+        "name": "Stats total time"
+      },
+      "stats_type": {
+        "name": "Stats type"
+      },
+      "wifi_ip": {
+        "name": "Ip"
```

Comment:
```suggestion
        "name": "IP address"
```

Reply:
Can I also use `"[%key:common::config_flow::data::ip%]"` even when it's not a config flow?

Reply:
I think we don't allow that, not 100% sure tho